linera-service/src/database_tool.rs

```diff
@@ -192,3 +182,21 @@ async fn main() -> Result<(), anyhow::Error> {
     tracing::info!("Successful execution of linera-db");
     Ok(())
 }
+
+#[tokio::main]
+async fn main() {
+    let env_filter = tracing_subscriber::EnvFilter::builder()
+        .with_default_directive(tracing_subscriber::filter::LevelFilter::INFO.into())
+        .from_env_lossy();
+    tracing_subscriber::fmt()
+        .with_writer(std::io::stderr)
+        .with_env_filter(env_filter)
+        .init();
+
+    let options = DatabaseToolOptions::from_args();
+    let error_code = match evaluate_options(options).await {
+        Ok(_) => 0,
+        Err(_) => 2,
```

Comment:
This swallows the error message in this case.

Reply:
Ok, I added a print statement for the error: `println!("Error is {:?}", msg);`.
Otherwise, I do not know how to do better than return values 0, 1, and 2.